samples/batch/transcription-enabled-storage/CreateTranscription/CreateTranscription.cs

```diff
@@ -0,0 +1,295 @@
+﻿// <copyright file="CreateTranscription.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace CreateTranscriptionFunction
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.IO;
+    using System.Linq;
+    using System.Net;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Connector;
+    using Microsoft.Azure.ServiceBus;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Extensions.Logging;
+    using Newtonsoft.Json;
+
+    public static class CreateTranscription
+    {
+        [FunctionName("CreateTranscription")]
+        public static void Run([ServiceBusTrigger("create_transcription_queue", Connection = "AzureServiceBus")]Message message, ILogger log)
+        {
+            if (log == null)
+            {
+                throw new ArgumentNullException(nameof(log));
+            }
+
+            if (message == null)
+            {
+                throw new ArgumentNullException(nameof(message));
+            }
+
+            log.LogInformation($"C# ServiceBus queue trigger function processed message: {message.Label}");
+            log.LogInformation($"Received message: SequenceNumber:{message.SystemProperties.SequenceNumber} Body:{Encoding.UTF8.GetString(message.Body)}");
+
+            var location = TranscribeMessage(message, log).Result;
+            log.LogInformation("Request Created, Guid: " + location);
+        }
+
+        internal static async Task<string> TranscribeMessage(Message message, ILogger log)
```

Comment:
message is a very generic name, I will make it more specific

Reply:
renamed it